src/many-modules/src/_4_events.rs

```diff
@@ -731,31 +742,34 @@ mod test {
             }
 
             #[test]
-            fn submit_submit_send(memo in "\\PC*", memo2 in "\\PC*", amount: u64) {
+            fn submit_submit_send(memo in string_regex("[A-Za-z0-9\\., ]{0,4000}").unwrap(), memo2 in string_regex("[A-Za-z0-9\\., ]{0,4000}").unwrap(), amount: u64) {
+                let memo = memo.try_into().unwrap();
+                let memo2 = memo2.try_into().unwrap();
                 _assert_serde(
                     _create_event_info(memo, vec![],
-                        AccountMultisigTransaction::AccountMultisigSubmit(
```

Comment:
Where did that rustfmt come from?

Reply:
Maybe the previous version didn't run rustfmt somehow?